contributors/design-proposals/kubelet-authorizer.md

```diff
@@ -79,37 +79,30 @@ This group and user name format match the identity created for each kubelet as p
 
 ## Node authorizer
 
-A new node authorizer will be inserted into the authorization chain:
-* API server authorizer (existing, authorizes "loopback" API clients used by components within the API server)
-* Node authorizer (new)
-* User-configured authorizers... (e.g. ABAC, RBAC, Webhook)
+A new node authorization mode (`Node`) will be made available for use in combination 
+with other authorization modes (for example `--authorization-mode=Node,RBAC`).
 
 The node authorizer does the following:
 1. If a request is not from a node (`IdentifyNode()` returns isNode=false), reject
-2. If a request is not allowed by the rules in the default `system:node` cluster rule, reject
-3. If a specific node cannot be identified (`IdentifyNode()` returns nodeName=""):
-    * If in compatibility-mode (default), allow. This lets nodes that don't use node-specific identities continue to work with the broad authorization rules in step 2.
-    * If in strict-mode, reject. This lets deployments that provision all nodes with individual identities to indicate that only identifiable nodes should be allowed.
-4. If a request is for a secret, configmap, persistent volume or persistent volume claim, reject unless the verb is `get`, and the requested object is related to the requesting node:
-    
-    * node -> pod
-    * node -> pod -> secret
-    * node -> pod -> configmap
-    * node -> pod -> pvc
-    * node -> pod -> pvc -> pv
-    * node -> pod -> pvc -> pv -> secret
-5. For other resources, allow
+2. If a specific node cannot be identified (`IdentifyNode()` returns nodeName=""), reject
+3. If a request is for a secret, configmap, persistent volume or persistent volume claim, reject unless the verb is `get`, and the requested object is related to the requesting node:
+
+    * node <-pod
+    * node <-pod-> secret
+    * node <-pod-> configmap
+    * node <-pod-> pvc
+    * node <-pod-> pvc <-pv
```

Comment:
nit: I'm confused by the arrow directions here. Isn't it the `pvc` that references the `pv`?